frontend/app/src/components/Sidebar/styled-components.ts

```diff
@@ -139,8 +148,6 @@ export const StyledSidebarHeaderContainer = styled.div(({ theme }) => ({
   display: "flex",
   justifyContent: "space-between",
   alignItems: "center",
-  paddingLeft: getSidebarHorizontalSpacing(theme),
-  paddingRight: getSidebarHorizontalSpacing(theme),
```

Comment:
We are moving this to `StyledSidebarContent` instead of applying the padding to all individual children in the sidebar.